backend/private-graph/graph/schema.resolvers.go

```diff
@@ -620,6 +620,23 @@ func (r *mutationResolver) EditWorkspace(ctx context.Context, id int, name *stri
 	return workspace, nil
 }
 
+// EditWorkspaceSettings is the resolver for the editWorkspaceSettings field.
+func (r *mutationResolver) EditWorkspaceSettings(ctx context.Context, workspaceID int, aiInsights *bool) (*model.AllWorkspaceSettings, error) {
+	_, err := r.isAdminInWorkspace(ctx, workspaceID)
+	if err != nil {
+		return nil, e.Wrap(err, "error querying workspace")
```

Comment:
we're trying to get rid of `e.Wrap` calls in general because we already have the error stacktraces. 
```suggestion
		return nil, err
```